include/ametsuchi/table/fixed_table.h

```diff
@@ -29,19 +29,21 @@
 namespace ametsuchi {
 namespace table {
 
+#define FILE_PREFIX 0
+
 template <typename T>
 class FixedTable {
  public:
-  class ForwardIterator;
+  class BidirIterator;
 
   using Record = BaseRecord<T>;
 
   FixedTable(const std::string &path);
 
-  void append(const T &data);
-  // void append(const T &&data);
+  file::offset_t append(const T &data);
+  file::offset_t append(const T &&data);
 
-  void appendBatch(const std::vector<T> &data);
+  file::offset_t appendBatch(const std::vector<T> &data);
```

Comment:
This data is written in consecutive way so it's still possible to retrieve all info from table (see tests)